cmd/satellite-object-storage-plugin/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM gcr.io/distroless/static:latest
+LABEL maintainers="Kubernetes Authors"
+LABEL description="Satellite Object Storage Plugin"
+
+COPY ./bin/satellite-object-storage-plugin satellite-object-storage-plugin
+ENTRYPOINT ["/satellite-object-storage-plugin"]
```

Comment:
we are planing to use single image for controller and node server.
Need to bundle the rclone, s3fs binaries along with dependencies.

Reply:
adding s3fs binary